palette/src/xyz.rs

```diff
@@ -246,9 +246,9 @@ where
     Wp: WhitePoint,
 {
     fn from_color_unclamped(color: Lab<Wp, T>) -> Self {
-        let y = (color.l + from_f64(16.0)) / from_f64(116.0);
-        let x = y + (color.a / from_f64(500.0));
-        let z = y - (color.b / from_f64(200.0));
+        let y = (color.l + from_f64(16.0)) * from_f64::<T>(116.0).recip();
+        let x = y + (color.a * from_f64::<T>(500.0).recip());
+        let z = y - (color.b * from_f64::<T>(200.0).recip());
```

Comment:
Same concept as the Srgb into linear optimization. This had the most dramatic effect, doubled the throughput.

```
Cie family/lab to xyz   time:   [861.87 ns 864.43 ns 867.12 ns]
                        thrpt:  [161.45 Melem/s 161.96 Melem/s 162.44 Melem/s]
                 change:
                        time:   [-34.197% -33.915% -33.587%] (p = 0.00 < 0.05)
                        thrpt:  [+50.573% +51.321% +51.968%]
```

Reply:
It's interesting how these have an effect when it's essentially the same thing. I'm aware that division is usually relatively slow, so I would understand the case with the determinant when the number of divisions is reduced. In this case you seem to add an instruction, but maybe there's a faster instruction for `1 / x` specifically.

Reply:
I was thinking maybe it helped the compiler see a better path for evaluating the values at compile time and pipelining instructions. In most other cases, trying to do this had no effect.

Reply:
Compilers are strange beings.